### PR TITLE
Update .upptimerc.yml - Add cherterafrica  Media

### PR DESCRIPTION
**Pull request by thepsalmist**
Add cherterafrica  Media

### CODE REVIEW

File: .upptimerc.yml [hunk 1]
```diff
@@ -48,6 +48,10 @@ sites:
     url: https://charter.africa
     icon: https://charter.africa/images/charter-logo.svg
 
+  - name: charterAFRICA - Media
+    url: https://charterafrica.prod.codeforafrica.org/media/charter-logo.svg
+    icon: https://charterafrica.prod.codeforafrica.org/media/charter-logo.svg
+
```
Comment (kilemensi) [inline .upptimerc.yml:54]: 1. Does this file come from S3 or is it the ones we have in code i.e. `public` folder?
2. Why are we not use _charter.africa_ domain i.e. `https://charter.africa/media/charter-logo.svg`?

**Review by kilemensi — APPROVED** [review]

File: .upptimerc.yml (hunk 1 above)
Comment (thepsalmist) [reply .upptimerc.yml:54]: @kilemensi

1. Does this file come from S3 or is it the ones we have in code i.e. public folder?

This should be served from S3, just replicated on `charterafrica-dev`, disabling S3 returns 404

2.Why are we not use charter.africa domain i.e. https://charter.africa/media/charter-logo.svg?

The actual served request URL is `charterafrica.prod`, though the `charter.africa` resolves to the same, unless there's a preference for the later

Comment (thepsalmist) [reply .upptimerc.yml:54]: Actually all media files are served from S3

Comment (kilemensi) [reply .upptimerc.yml:54]: ... yeah, weird seeing codeforafrica.org in the URL there since this site is for a separate org.

**Pull request merged**